frontend/public/components/factory/table.tsx

```diff
@@ -112,7 +112,16 @@ const sorts = {
   },
 };
 
-const stateToProps = ({UI}, {data = [], defaultSortField = 'metadata.name', defaultSortFunc = undefined, filters = {}, loaded = false, reduxID = null, reduxIDs = null, staticFilters = [{}], rowFilters = []}) => {
+const stateToProps = ({UI}, {
+  data = [],
+  defaultSortField = 'metadata.name',
+  defaultSortFunc = undefined,
+  filters = {},
+  loaded = false,
+  reduxID = null,
+  reduxIDs = null,
+  staticFilters = [{}],
+  rowFilters = []}) => {
```

Comment:
not really sure what this means ?

Reply:
This is setting `rowFilters = []`, the closing `}` is a closing bracket for an object which is the second argument of `stateToProps` function.